integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/context/ProcessorContext.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.service.bridge.integration.tests.context;
```

Comment:
I added the `ProcessorContext` so it would be easier later for any additional information